services/dkg/pedersen/controller/action.go

```diff
@@ -316,6 +316,8 @@ func (a *RegisterHandlersAction) Execute(ctx node.Context) error {
 
 	ep := eproxy.NewDKG(mngr, dkg, proxykey)
 
+	// Register the proxy handlers
+	// DKG init
```

Comment:
I'm not sure I understand the DKG init comment

Reply:
Noemien told that it's here that the http request is connected to the corresponding function